pkg/kube/client.go

```diff
@@ -412,7 +412,6 @@ func newClientInternal(clientFactory util.Factory, revision string) (*client, er
 		return nil, err
 	}
 	c.dynamicInformer = xnsinformers.NewDynamicSharedInformerFactory(c.dynamic, resyncInterval)
-	c.dynamicInformer.SetNamespaces() // Maistra needs to start with an empty namespace set.
```

Comment:
There no longer any need for this, since this is done automatically when you register the informer as a listener in the member roll controller.